cirq/experiments/qubit_characterizations.py

```diff
@@ -0,0 +1,659 @@
+        return [(angle, prob) for angle, prob in zip(self._rabi_angles,
+                                                     self._excited_state_probs)]
+
+    def plot(self, **plot_kwargs: Any) -> None:
+        """Plots excited state probability vs the Rabi angle (angle of rotation
+        around the x-axis).
+
+        Args:
+            **plot_kwargs: Arguments to be passed to matplotlib.pyplot.plot.
+        """
+        fig = plt.figure()
+        plt.plot(self._rabi_angles, self._excited_state_probs, 'ro-',
+                 figure=fig, **plot_kwargs)
+        plt.xlabel(r"Rabi Angle (Radian)", figure=fig)
+        plt.ylabel('Excited State Probability', figure=fig)
+        fig.show()
+
+
+class RandomizedBenchMarkResult:
+    """Results from a randomized benchmarking experiment."""
+
+    def __init__(self, num_cliffords: Sequence[int],
+                 ground_state_probabilities: Sequence[float]):
+        """
+        Args:
+            num_cliffords: The different numbers of Cliffords in the RB
+                study.
+            ground_state_probabilities: The corresponding average ground state
+                probabilities.
+        """
+        self._num_cfds_seq = num_cliffords
+        self._gnd_state_probs = ground_state_probabilities
+
+    @property
+    def data(self) -> Sequence[Tuple[int, float]]:
+        """Returns a sequence of tuple pairs with the first item being a
+        number of Cliffords and the second item being the corresponding average
+        ground state probability.
+        """
+        return [(num, prob) for num, prob in zip(self._num_cfds_seq,
+                                                 self._gnd_state_probs)]
+
+    def plot(self, **plot_kwargs: Any) -> None:
+        """Plots the average ground state probability vs the number of
+        Cliffords in the RB study.
+
+        Args:
+            **plot_kwargs: Arguments to be passed to matplotlib.pyplot.plot.
+        """
+        fig = plt.figure()
+        plt.plot(self._num_cfds_seq, self._gnd_state_probs, 'ro-',
+                 figure=fig, **plot_kwargs)
+        plt.xlabel(r"Number of Cliffords", figure=fig)
+        plt.ylabel('Ground State Probability', figure=fig)
+        fig.show()
+
+
+class TomographyResult:
+    """Results from a state tomography experiment."""
+
+    def __init__(self, density_matrix: np.ndarray):
+        """
+        Args:
+            density_matrix: The density matrix obtained from tomography.
+        """
+        self._density_matrix = density_matrix
+
+    @property
+    def data(self) -> np.ndarray:
+        """Returns an n^2 by n^2 complex matrix representing the density
+        matrix of the n-qubit system.
+        """
+        return self._density_matrix
+
+    def plot(self) -> None:
+        """Plots the real and imaginary parts of the density matrix as two
+        3D bar plots.
+        """
+        fig_re, fig_imag = _plot_density_matrix(self._density_matrix)
+        fig_re.show()
```

Comment:
Show both figures simultaneously without one blocking the other, e.g. by merging the plots into one figure if necessary. When the user calls plot they should be seeing all the data right away.

Reply:
Done. The bar plots for the real&imaginary parts of the density matrix now fit into a single figure. The display looks fine on my computer. Let me know if it looks weird on your computer and I will figure out a better way to do it.